### PR TITLE
DOC-2525 add ListTutorial.cs to fork

### PR DESCRIPTION
**Pull request by andy-stark-redis**

### CODE REVIEW

**Comment by andy-stark-redis** [comment]
Closes #213

File: tests/Doc/Doc.csproj [hunk 1]
```diff
@@ -22,4 +22,7 @@
     <ProjectReference Include="..\..\src\NRedisStack\NRedisStack.csproj" />
     <ProjectReference Include="..\..\tests\NRedisStack.Tests\NRedisStack.Tests.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
```
Comment (atakavci) [inline tests/Doc/Doc.csproj:26]: do we still need this ??

File: tests/Doc/xunit.runner.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "parallelizeAssembly": false,
```
Comment (atakavci) [inline tests/Doc/xunit.runner.json:2]: and these ?

**Review by atakavci — APPROVED** [review]
looks good !

**Pull request merged**